templates/_macros/vf_tiered-list.jinja

```diff
@@ -74,23 +74,27 @@
           {%- if is_list_full_width_on_tablet == true %}
             <div class="row">
```

Comment:
Just noticed one more problem

There is now an empty set of `row` and `col` divs to contain the first item's nonexistent HR.

Can we remove these divs entirely for the first item to keep the DOM a bit cleaner?

<img width="1412" alt="Screenshot 2025-05-29 at 10 10 36" src="https://github.com/user-attachments/assets/0adf2d65-aa05-4883-8caa-3338fcba79f7" />